src/query/expression/src/kernels/filter.rs

```diff
@@ -70,6 +72,50 @@ impl DataBlock {
         }
     }
 
+    pub fn split_filter_with_bitmap(self, bitmap: &Bitmap) -> Result<(DataBlock, DataBlock)> {
```

Comment:
Is it benefit in performance comparing to:
```rust
let true_rows = block.filter_with_bitmap(filter)?;
let false_rows = block.filter_with_bitmap(!filter)?;
```

Reply:
avoid data block clone, here is self, not &self.

Reply:
datablock clone is merely `Arc::clone`